docs/Configuration.md

```diff
@@ -2274,6 +2274,8 @@ After the worker has executed a test the memory usage of it is checked. If it ex
     - `G` / `GB` - Gigabytes
     - `GiB` - Gibibytes
 
+**NOTE:** [% based memory does not work on Linux CircleCI workers](https://github.com/facebook/jest/issues/11956#issuecomment-1212925677) due to incorrect system memory being reported.
```

Comment:
we should keep this, but I don't think we should do runtime checks - as mentioned this is the same as `--maxWorkers` which is only called out in the docs